packages/legend-extension-external-store-service/src/models/protocols/pure/ESService_PureProtocolProcessorPlugin.ts

```diff
@@ -281,71 +280,6 @@ export class ESService_PureProtocolProcessorPlugin
     ];
   }
 
-  V1_getExtraClassMappingSecondPassBuilders(): V1_ClassMappingSecondPassBuilder[] {
```

Comment:
just confirming you meant to delete this? seems like a lot of logic we dont need anymore ?

Reply:
yeah i asked him to do that because i was doing the same processing in first pass and second pass. I messed up a bit there while i did that

Reply:
yes this is not needed, this was duplicated code